packages/node/src/integrations/contextlines.ts

```diff
@@ -0,0 +1,138 @@
+import { getCurrentHub } from '@sentry/core';
+import { Event, EventProcessor, Integration } from '@sentry/types';
+import { addContextToFrame } from '@sentry/utils';
+import { readFile } from 'fs';
+import { LRUMap } from 'lru_map';
+
+import { NodeClient } from '../client';
+
+const FILE_CONTENT_CACHE = new LRUMap<string, string | null>(100);
```

Comment:
wonder now that this is in an integration we should ever make this configurable.

Reply:
Move it inside the integration class and allow max cache size config?

Reply:
yup, I leave it for you to make the call on, as nobodies ever requested this per say (as it is a pretty internal implementation detail)

Reply:
The integration constructor takes an object so it would be easy to add this in the future without breaking changes